Author: Giuseppe Iuculano <giuseppe@iuculano.it>
Description: Read logs from /var/log instead of /var/adm
--- a/README.chklastlog
+++ b/README.chklastlog
@@ -31,8 +31,8 @@ chklastlog - check lastlog-file for dele
    % cc -o chklastlog chklastlog.c 
 
 
-   To run chklastlog you need read permission on the files /var/adm/wtmp
-   and /var/adm/lastlogin. Normally these files are world-readable and
+   To run chklastlog you need read permission on the files /var/log/wtmp
+   and /var/log/lastlog. Normally these files are world-readable and
    no special privileges are required to run the checker.
 
    The following is an example of the output of chklastlog.
--- a/README.chkwtmp
+++ b/README.chkwtmp
@@ -30,7 +30,7 @@ chkwtmp - check wtmp-file for overwritte
    To create chkwtmp under SunOS 4.x, type:
    % cc -o chkwtmp chkwtmp.c 
 
-   To run chkwtmp you need read permission on the file /var/adm/wtmp.
+   To run chkwtmp you need read permission on the file /var/log/wtmp.
    Normally this file is world-readable and no special privileges are 
    required to run the checker. 
 
--- a/check_wtmpx.c
+++ b/check_wtmpx.c
@@ -30,8 +30,8 @@ int main () { return 0; }
 #include <fcntl.h>
 #include <unistd.h>
 
-#define WTMP_FILENAME  "/var/adm/wtmp"
-#define WTMPX_FILENAME "/var/adm/wtmpx"
+#define WTMP_FILENAME  "/var/log/wtmp"
+#define WTMPX_FILENAME "/var/log/wtmpx"
 
 
 struct file_utmp_entry
--- a/chklastlog.c
+++ b/chklastlog.c
@@ -61,10 +61,10 @@
 #define LASTLOG_FILENAME "/var/log/lastlog"
 #endif
 #ifndef WTMP_FILENAME
-#define WTMP_FILENAME "/var/adm/wtmp"
+#define WTMP_FILENAME "/var/log/wtmp"
 #endif
 #ifndef LASTLOG_FILENAME
-#define LASTLOG_FILENAME "/var/adm/lastlog"
+#define LASTLOG_FILENAME "/var/log/lastlog"
 #endif
 
 #define TRUE 1L
--- a/chkutmp.c
+++ b/chkutmp.c
@@ -48,7 +48,7 @@ int main () { return 0; }
 #define MAXLENGTH 256
 #define UT_PIDSIZE 12
 #if defined(__sun)
-#define UTMP "/var/adm/utmpx"
+#define UTMP "/var/log/utmpx"
 #define UT_LINESIZE 12
 #define UT_NAMESIZE 8
 #define PS_CMD 0
--- a/chkwtmp.c
+++ b/chkwtmp.c
@@ -36,7 +36,7 @@
 #define WTMP_FILENAME "/var/log/wtmp"
 #else
 #ifndef WTMP_FILENAME
-#define WTMP_FILENAME "/var/adm/wtmp"
+#define WTMP_FILENAME "/var/log/wtmp"
 #endif
 #endif
 
